index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="header-bg">
+            <img class="Ellipse12" src="images/Ellipse 12.png" alt="">
+            <img class="Ellipse13" src="images/Ellipse 13.png" alt="">
+            <img class="Ellipse4" src="images/Ellipse 4.png" alt="">
+            <img class="Rectangle5" src="images/Rectangle 5.png" alt="">
+            <img class="Ellipse3" src="images/Ellipse 3.png" alt="">
+            <img class="Rectangle4" src="images/Rectangle 4.png" alt="">
+            <img class="Rectangle7" src="images/Rectangle 7.png" alt="">
+            <img class="Ellipse5" src="images/Ellipse 5.png" alt="">
+            <img class="Ellipse11" src="images/Ellipse 11.png" alt="">
+            <img class="Rectangle6" src="images/Rectangle 6.png" alt="">
+        </div>
+        <section class="header">
+            <label class="logo" >My Logo</label>
+            <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </section>
+
+        <section class="main">
+            <div class="Text">
+                <div class="header2">
+                    <h1>I'm Alex Waweru. Glad to see you!</h1>
+                </div>
+                <div class="primary-text1">
+                    <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+                        don't hesitate to contact me.
+                       </p>
+                </div>
+                <div class="primary-Text">
+                    <p>LET'S CONNECT</p>
+                    <label class="icons">
+                        <img class="twitter" src="images/twitter.svg" alt="twitter">
+                        <img class="Vector1" src="images/Vector.png" alt="Vector1">
+                        <img class="Vector2" src="images/Vector (1).png" alt="Vector2">
+                        <img class="medium" src="images/medium.svg" alt="medium">
+                        <img class="github" src="images/github.svg" alt="github">
+                        <img class="angellist" src="images/angellist.svg" alt="angellist">
+                        
+                    </label>
```

Comment:
- [ ] Kindly, ensure that the `Let's Connect` has (`a` tag) or a (`button` tag) since it's a call to action so it can create a hyperlink to other web pages, files, and locations within the same page, email addresses, etc. :+1: 
- You also need to ensure that it has the right `font-family`, `font-size`, and `color` as well as margins and paddings (top, bottom, left, right) defined in the design. :100: 
- [ ] Please ensure that the list of icons uses a (`ul` tag) to group them instead of the (`label` tag), and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`.
- You also need you to ensure that it has the right margins and paddings (top, bottom, left, right) defined in the design. :100: